src/eduid/webapp/idp/login.py

```diff
@@ -92,6 +97,19 @@ def __str__(self):
 
 def login_next_step(ticket: LoginContext, sso_session: Optional[SSOSession], template_mode: bool = False) -> NextResult:
     """ The main state machine for the login flow(s). """
+    if current_app.conf.allow_other_device_logins:
+        if ticket.is_other_device == 1:
```

Comment:
While "initiator" and "authenticator" or similar are more descriptive for the first-time code reader, I actually think that it will slow down working with the code/reading logs for us. For me at least, one and two parses much faster in my brain.

I agree is_first_device and is_second_device with pure boolean return types might be good though. Not sure if it is better than what I made this far, but I agree it isn't worse :).

Reply:
For you it might be better, I can probably learn it but I think things can go horribly wrong if 1 and 2 is mixed up somewhere. So it would be safer to use a explicit name.

Reply:
As an outsider, I view `1` and `2` as magic values and I would also prefer they were named.

Reply:
Ok I'm prepared to go with it and use enums instead of integers. You have a good point about magic values.

However, I still don't think use_other_first is a better name for the view than use_other_1, and device_first.py a better name than device_1.py and OtherDevice.first of type FirstDeviceData to be better than OtherDevice.device1 of type Device1Data :).

I am prepared to go with a majority of the three of us overruling the minority though.

Reply:
The magic numbers are gone. Talked more with Johan and went with separate is_other_device_1 and _2 functions returning booleans.